en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md

```diff
@@ -1012,10 +1012,72 @@ Follow the instructions given below if you have used a **custom log4j component*
 
 ### Migrating encrypted passwords
 
-To migrate the encrypted passwords from EI 6.1.0, you need to first obtain the plain-text passwords. Once you have them, follow the normal procedure of encrypting secrets in the Micro Integrator. See [Encrypt Secrets]({{base_path}}/install-and-setup/setup/mi-setup/security/encrypting_plain_text) for instructions.
+To migrate the encrypted passwords from EI 6.1.0, you need to first re-encrypt (using OAEP) all the encrypted information 
+in the database. Then, you can obtain the plain-text passwords by following the normal procedure of encrypting secrets 
+in the Micro Integrator. See [Encrypting Secrets]({{base_path}}/install-and-setup/setup/mi-setup/security/encrypting_plain_text) 
+for instructions.
 
-In case you need to obtain the plain-text passwords by decrypting the encrypted passwords in the EI 6.1.0,
-you can use the [password decryption tool](https://github.com/wso2-docs/WSO2_EI/tree/master/migration-client).
+!!! Info API-M 4.0.0 uses OAEP for data encryption in addition to the RSA algorithm 
```

Comment:
Does this render correctly when you generate the docs locally?

Reply:
fixed